### DISCUSSION TITLE
New preset values

### DISCUSSION THREAD

**Discussion by rstockm** [TheLastBen/fast-stable-diffusion#1878] [Q&A]
Some days (?) ago, some preset values in the training section got updated with new preset values:

UNet_Training_Steps from 500 to 1500 (per 10 images)
UNet_Learning_Rate from (?) to 2e-6 (lowest)

These changes are quite significant, and I would be interested: why and to what effect? And is there a dependency to the "no more than 20 pictures for a portrait" mantra?

And then:

Text_Encoder_Learning_Rate = 1e-6
But the help text reads: keep it low to avoid overfitting (1e-6 is higher than 4e-7)
So - why not changing the preset to a lower setting, what are the pros/cons?

Thanks in advance.

**Comment by TheLastBen** [comment]
I changed the steps from 500 to 1500 because I lowered the learning rate from 1e-5 to 2e-6 in the default settings to make sure users won't face excessive overfitting.

the value 1e-6 seems okay for the text encoder.

Upping the values will accelerate the training but could break the model, so one would have to experiment with different settings.
